From 4824766a059389fae7136f840e8cad8d62d4c1a9 Mon Sep 17 00:00:00 2001
From: Alex Cruz <du.alexcruz@gmail.com>
Date: Mon, 9 Apr 2018 18:51:28 -0400
Subject: [PATCH 260/296] Fix text cutting off during shutdown animation

No better way to explain it than through a picture

Google some how did not account for what I like to call 'Can't see
shit mode' and so this happened :p
---
 core/res/res/layout/shutdown_dialog.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/res/res/layout/shutdown_dialog.xml b/core/res/res/layout/shutdown_dialog.xml
index 398bfb1824c..c3fc51e096c 100644
--- a/core/res/res/layout/shutdown_dialog.xml
+++ b/core/res/res/layout/shutdown_dialog.xml
@@ -29,7 +29,7 @@
     <TextView
         android:id="@id/text1"
         android:layout_width="wrap_content"
-        android:layout_height="32dp"
+        android:layout_height="wrap_content"
         android:text="@string/shutdown_progress"
         android:textDirection="locale"
         android:textSize="24sp"
-- 
2.11.0

